config/crd/bases/postgres-operator.crunchydata.com_pgadmins.yaml

```diff
@@ -1448,6 +1448,25 @@ spec:
                   not show up here.
                 items:
                   properties:
+                    passwordRef:
+                      description: A reference to the secret that holds the user's
+                        password.
+                      properties:
+                        key:
+                          description: The key of the secret to select from.  Must
+                            be a valid secret key.
+                          type: string
+                        name:
+                          description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                            TODO: Add other useful fields. apiVersion, kind, uid?'
```

Comment:
:wrench: We should remove this TODO. We have tooling in place for this message in other parts of the CRD, I think you'll just need to add another entry in `build/crd/pgadmins/todos.yaml`.

Reply:
Did we not catch this in any actions? :thinking:

Reply:
I don't see anything in our actions that should have caught it, but probably makes sense for something to be added.